model/bin/make_makefile.sh

```diff
@@ -327,11 +332,6 @@
                ID='rotated grid'
                TS='RTD'
                OK='RTD' ;;
-#sort:arctic:
```

Comment:
the corresponding section in w3_new can also be removed for arctic

Reply:
RTD option is independent of SMC grid though they share some subroutines.

Reply:
The RTD option is still there with rotag, but the arctic and ARC switch are removed, correct?  If so, the "arctic" option could be removed from w3_new as well as being removed from make_makefile.sh

Reply:
Yes, the arctic option in w3_new could be removed as the ARC switch is removed.

Reply:
Can you remove the "arctic" section from w3_new, please @ukmo-jianguo-li

Reply:
thanks @ukmo-jianguo-li